controller/user.js

```diff
@@ -32,7 +35,7 @@ async function getAll(req, res) {
     code: 0,
     msg: "",
     data: {
-      users: users.map(user => buildUserJson(user))
+      users: users.map(user => formatUser(user))
```

Comment:
rename function to `getAllUsers`

It seems that the function is not needed. Please check the API docs. Comment the function.